src/Frontend/config/initialState.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+export default {
+  menu: {
+    items: []
+  },
+  user: [],
+  locales: {
+    "de": "Deutsch",
+    "en": "English"
+  },
+  currentLocale: 'en'
```

Comment:
I'm not sure here what to do. Each level of the tree can be represented by a reducer, but I'm not sure if we want a `localesReducer` and a `currentLocaleReducer`.
I'd rather build a single `locales` object which contains another object called `available` for the locale object(`{ de: Deutsch, ... }`) and a single, scalar field called `current`. Or am I missing a reason why `locales` and `currentLocale` should be separated?